menyer/enmeny/style1.css

```diff
@@ -19,34 +19,18 @@ header {
     box-shadow: 13px 13px 8px 8px magenta;
 }
 
-.skinkis, .fruktis, .julis, .paris, .kottis, .cremis {
+.matgrej {
     border-style: dotted;
     border-width: 3px;
     border-radius: 30px;
+    margin-bottom: 20px;
 }
 
-.huvudratter {
-    background-color: aqua;
-    padding: 20px;
-    margin: 200px;
-    border-radius: 30px;
-    color: blueviolet;
-    box-shadow: 13px 13px 8px 8px magenta;
-}
-
-.efterratter {
-    background-color: aqua;
-    padding: 20px;
-    margin: 200px;
-    border-radius: 30px;
-    color: blueviolet;
-    box-shadow: 13px 13px 8px 8px magenta;
-}
-
-.forratterh2, .huvudratterh2, .efterratt {
+.h2rub {
     color: darkmagenta;
     font-size: 33px;
 }
+
 .besk, .skinkmackah3, .ettparon, .fruktsallad, .julbord, .kottbit, .creme {
```

Comment:
If so many selectors have the same style change is the selectors to something more general and apply it to all cases.

Perhaps `.matgrej-text`